client/src/Components/Pages/Questionnaire/PersonalDetails2/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+
+import './style.css'
+import '../style.css'
+
+const options = [
+
+    { value: '1', label: '1' },
+    { value: '2', label: '2' },
+    { value: '3', label: '3' },
+    { value: '4', label: '4' },
+    { value: '5', label: '5' },
+ 
+  ];
+
+class PersonalDetails2 extends Component {
+
+    state = {
+        selectedOption: null, 
+    }
+
+    handleChange = (selectedOption) => {
+
+        this.setState();
+        
+      }
+
+    Selector = () => {
+       return <Select
+        onChange={this.handleChange}
+        value={this.selectedOption}
```

Comment:
Here you'll need `this.state.selectedOption` or something similar, not `this.selectedOption`.

Reply:
What is the difference?

Reply:
I think you'll be storing the user's selected option in the component's state (in `this.state.selectedOption`), once you finish out the handleChange function up on line 23-26 (by calling `this.setState({ selectedOption: selectedOption })` or, a shorter version which does the same thing, `this.setState({ selectedOption })`).

You basically want this `<Select>` to both display the current value from a component state attribute, and also write to that same component state attribute, to make everything work (and have the component re-render when things change).  It might appear to be working right now because handleChange isn't quite finished.  Let me know if that makes sense, I can explain it more if not!